ironfish-rust/src/transaction/mod.rs

```diff
@@ -334,15 +329,18 @@ impl ProposedTransaction {
     }
 }
 
+impl Default for ProposedTransaction {
```

Comment:
Not sure why clippy didn't complain about this before, but rust-analyzer can auto-generate this for you, nice.